### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by occupant**
Merge updated from Develop branch:
Changes:
- add default region settings for tabs, main content, breadcrumbs - ca27d75
- sticky nav style tweaks -  18ac6e2
- correct ubc.ca domain protocol (http -> https) -  a21723d
- adjustments to the message template - c603477
- fix for twig template suggestion appearing in incorrect order - 088e72c
- google+ social network buttons repurposed to instagram - d64cd49
- update node-sass to latest version - 914e27b

### CODE REVIEW

File: config/install/block.block.breadcrumbs.yml [hunk 1]
```diff
@@ -18,4 +18,9 @@ settings:
   label: Breadcrumbs
   provider: system
   label_display: '0'
-visibility: {  }
+visibility:
```
Comment (occupant) [inline config/install/block.block.breadcrumbs.yml:21]: Explanation: add some default visibility settings (excluded from front page)

File: config/install/block.block.mainpagecontent.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+langcode: en
```
Comment (occupant) [inline config/install/block.block.mainpagecontent.yml:1]: Explanation: assigning default region settings

File: config/install/block.block.tabs.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+langcode: en
```
Comment (occupant) [inline config/install/block.block.tabs.yml:1]: Explanation: assigning default region settings

File: config/install/galactus.settings.yml [hunk 4]
```diff
@@ -19,6 +19,7 @@ clf_unitname: 'Unit Name'
 clf_theme_colour: 'bw'
 clf_layout: 'full'
 clf_override: 0
+clf_source: ''
```
Comment (occupant) [inline config/install/galactus.settings.yml:22]: Explanation: add default variable settings on install

File: css/navigation.sticky.css [hunk 5]
```diff
@@ -1,6 +1,6 @@
 /**
  * @file
- * Placeholder
+ * Styles for sticky navigation
```
Comment (occupant) [inline css/navigation.sticky.css:3]: Explanation: update comment and add explicit z-index

File: galactus.libraries.yml [hunk 6]
```diff
@@ -6,40 +6,53 @@ global:
     theme:
       css/clf.drupal.css: {}
   js:
-    https://cdn.ubc.ca/clf/7.0.4/js/ubc-clf.min.js: {}
     js/lib/bootstrap/alert.js: {}
   dependencies:
     - core/jquery
 
 ### CDN CLF Versions ###
+cdn-clf-js:
```
Comment (occupant) [inline galactus.libraries.yml:14]: Explanation: add CDN js as a separate library to simplify and reduce duplication

File: galactus.libraries.yml [hunk 7]
```diff
@@ -115,6 +142,21 @@ clf-sw-gw:
       https://cdn.ubc.ca/clf/7.0.4-minimal/css/minimal-clf-7.0.4-gw.css: { type: external, minified: true }
     theme:
       css/clf.social-icons.css: {}
+  dependencies:
+    - galactus/cdn-clf-js
+
+# Local Version of CLF (all variants)
```
Comment (occupant) [inline galactus.libraries.yml:148]: Explanation: adds local CLF library assets

File: galactus.theme [hunk 8]
```diff
@@ -11,6 +11,7 @@ use Drupal\Core\Form\FormStateInterface;
  * Implements hook_page_attachments_alter().
  */
 function galactus_page_attachments_alter(array &$page) {
+  $clf_source = theme_get_setting('clf_source');
```
Comment (occupant) [inline galactus.theme:14]: Explanation: adds theme setting as variable

File: galactus.theme [hunk 9]
```diff
@@ -23,9 +24,12 @@ function galactus_page_attachments_alter(array &$page) {
   }
 
   // Switch between minimal and full CLF libraries.
-  if ($use_minimal_clf) {
+  if ($clf_source == 'min' || $use_minimal_clf) {
```
Comment (occupant) [inline galactus.theme:27]: Explanation: check for either legacy setting for minimal CLF or the new theme source setting to load the CDN version of the minimal CLF

File: galactus.theme [hunk 10]
```diff
@@ -23,9 +24,12 @@ function galactus_page_attachments_alter(array &$page) {
...
     $page['#attached']['library'][] = 'galactus/clf-' . $clf_layout_id . '-' . $clf_theme_color;
   }
+  elseif ($clf_source == 'local') {
```
Comment (occupant) [inline galactus.theme:30]: Explanation: Load the local CLF files if that option is selected

File: galactus.theme [hunk 11]
```diff
@@ -196,6 +200,10 @@ function galactus_preprocess_html(&$variables) {
     $variables['#attached']['library'][] = 'galactus/clf-fontawesome';
   }
 
+  // Add a class to define the active CLF theme colour
```
Comment (occupant) [inline galactus.theme:203]: Explanation: use the theme colour setting to add a new class to the body tag for use in our local CLF CSS file

File: galactus.theme [hunk 12]
```diff
@@ -373,11 +381,7 @@ function galactus_form_alter(&$form, FormStateInterface $form_state, $form_id) {
  */
 function galactus_theme_suggestions_page_alter(array &$suggestions, array $variables) {
   // Get current node.
-  $node = \Drupal::request()->attributes->get('node');
-  if ($node) {
-    // Add the Node Type as a template suggestion.
-    if ($node->getType()) {
-      $suggestions[] = 'page__node_type__' . $node->getType();
-    }
+  if (($node = \Drupal::request()->attributes->get('node')) && (strpos($_SERVER['REQUEST_URI'], "revisions") == false)) {
```
Comment (occupant) [inline galactus.theme:384]: Explanation: update function to correctly load template suggestions when previewing content with revisions.

File: templates/layout/page.html.twig [hunk 13]
```diff
@@ -73,7 +73,7 @@
         <div class="container">
         {% endif %}
         <div id="ubc7-search-box">
-          <form class="form-search" method="get" action="http://www.ubc.ca/search/refine/" role="search">
+          <form class="form-search" method="get" action="https://www.ubc.ca/search/refine/" role="search">
```
Comment (occupant) [inline templates/layout/page.html.twig:76]: Explanation: fix protocol

File: templates/misc/status-messages.html.twig [hunk 14]
```diff
@@ -33,10 +33,7 @@
       'alert-' ~ type,
     ]
   %}
-  <div role="contentinfo" aria-label="{{ status_headings[type] }}"{{ attributes.addClass(classes)|without('role', 'aria-label') }}>
-    {#% if type == 'error' %}
-      <div role="alert">
-    {% endif %#}
+  <div role="contentinfo" aria-label="{{ status_headings[type] }}"{{ attributes.setAttribute('class', classes)|without('role', 'aria-label') }}>
```
Comment (occupant) [inline templates/misc/status-messages.html.twig:36]: Explanation: remove unused comments in code

File: theme-settings.php [hunk 15]
```diff
@@ -86,10 +86,22 @@ function galactus_form_system_theme_settings_alter(&$form, FormStateInterface &$
   $form['general']['clf_override'] = [
     '#type' => 'checkbox',
     '#title' => t('<strong>Use Minimal CLF</strong>'),
-    '#description' => t('Use an external, minimal copy of the CLS css files.<br><strong>Warning</strong>: Advanced users only. This will require creating and adding your own css for non-clf regions of the site, including navigation.'),
+    '#description' => t('Use an external, minimal copy of the CLS css files.<br><strong>Warning</strong>: Advanced users only. This will require creating and adding your own css for non-clf regions of the site, including navigation.<br><small><strong>Replaced by <i>CLF Version and Source</i> setting, kept for legacy purposes.</strong></small>'),
```
Comment (occupant) [inline theme-settings.php:89]: Explanation: update setting description to indicate deprecation for new setting.

File: theme-settings.php [hunk 16]
```diff
@@ -86,10 +86,22 @@ function galactus_form_system_theme_settings_alter(&$form, FormStateInterface &$
...
     '#default_value' => theme_get_setting('clf_override'),
   ];
 
+  $form['general']['clf_source'] = [
```
Comment (occupant) [inline theme-settings.php:93]: Explanation: add new theme setting to define the source of the CLF (CDN full package, CDN minimal package, Local)

**Comment by joelpittet** [comment]
I merged things back from master... that didn't do what I expected... but now they are in sync 😳

**Pull request closed without merging**